go/stats/main.go

```diff
@@ -376,27 +359,6 @@ func main() {
 	}
 }
 
-func tailFile(stats pfconfigdriver.PfStats, config tail.Config, done chan bool) {
```

Comment:
I was more thinking of moving this logic to ProcessMetricConfig when the type=tail_file so that we can still keep doing this

Reply:
I don't understand the purpose of this metric? What value will we pass to statsd with its use?

Reply:
This actually tails a log file and checks for some specific patterns, when they do match, we increment a counter in statsd to know how many times that pattern matched.

The best example I could give would be to match on ERROR in a specific log to see how many errors there are in it.

Since we're not using it and I'm sure @extrafu would like to see that in ASAP, then lets scratch that and implement it when needed